src/views/SignTransactionLedger.vue

```diff
@@ -153,8 +166,22 @@ export default class SignTransactionLedger extends Vue {
                 label: this.rpcState.origin.split('://')[1],
                 image: checkoutRequest.shopLogoUrl,
             };
+        } else if (this.request.kind === RequestType.CASHLINK) {
+            // coming from cashlink create
+            if (!this.cashlink) {
+                // happens if user reloads the page
+                this._back();
+                return;
+            }
+            senderUserFriendlyAddress = this.$store.state.activeUserFriendlyAddress;
+
+            this.recipientDetails = {
+                address: this.cashlink!.address.toUserFriendlyAddress(),
```

Comment:
Since there is a `return` in the check for `this.cashlink` above, I think the `!` might not be necessary here anymore:
```suggestion
                address: this.cashlink.address.toUserFriendlyAddress(),
```